docs/docs/authenticate/azure-ad.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: Azure Active Directory
+nav_order: 3
+parent: Authenticate
+description: >
+  This page contains instructions on integrating Azure AD as
+  an identity provider for Firezone
+---
+---
+
+Firezone supports Single Sign-On (SSO) using Azure Active Directory through the generic
+OIDC connector. This guide will walk you through how to obtain the following
+config settings required for the integration:
+
+1. `discovery_document_uri`: This URL returns a JSON with information to
+construct a request to the OpenID server.
+1. `client_id`: The client ID of the application.
+1. `client_secret`: The client secret of the application.
+1. `redirect_uri`: Instructs OIDC identity provider where the response to the
+request should be sent.
+1. `response_type`: Set to `code`.
+1. `scope`: The permissions required by Firezone.
```

Comment:
suggested technical clarification:

```
1. `scope`: [OIDC scopes](https://openid.net/specs/openid-connect-basic-1_0.html#Scopes) to obtain from your OIDC provider. This should be set to `openid email` to provide Firezone with the user's email in the returned claims.

Reply:
Nevermind, we can leave `profile` in there as it's fairly common and will be useful for sending outbound emails.